modules/BuildUtilities/src/server/python/zzz-config-to-envs.py

```diff
@@ -0,0 +1,146 @@
+# Process configs
```

Comment:
It would be helpful somewhere to document what this script is for. As I understand it is not directly used during startup - it is just a utility for admins to convert their legacy config files to environment variables, right?
Some explanation of this should either be added to the top of this script, to the README, or both.

Reply:
This script needs a lot of work. I might spin out another ticket for that instead of resolving all the issues as part of this merge. I will actually remove it here.

Reply:
Fair enough. I'm fine with you leaving it in as long as there's another ticket spun out for cleaning it up. I just picture it prompting a lot of questions if you point an SA at it right now.